### DISCUSSION TITLE
QuickTranslate

### DISCUSSION THREAD

**Discussion by karubn** [MurageKabui/CL-QuickTranslate#1] [General]
Hello Mr. Kabue Murage, first of all, the program we coded and prepared CL-QuickTranslate is very nice, I needed a program that translates a pdf word exell or text file (txt) directly with the command line interface, just like I was looking for, but the problem is that CL-QuickTranslate is the program that translates the text file over 5 pages. The text file is empty. I wanted to ask you this???? I added the sample file to the attachment section ms5.txt
[ms5.txt](https://github.com/KabueMurage/CL-QuickTranslate/files/9087407/ms5.txt)

**Comment by karubn** [comment]
Or I can explain as follows, I shared a zip file in it, ms4.txt translates, but ms5.txt translation is a blank page, so I would appreciate it if you are interested.
[CL-QuickTranslate-master.zip](https://github.com/KabueMurage/CL-QuickTranslate/files/9087467/CL-QuickTranslate-master.zip)

**Reply by MurageKabui** [reply]
Thanks for feedback @karubn! Unfortunately there isn't much at [cli.bat](https://github.com/KabueMurage/CL-QuickTranslate/files/9087467/CL-QuickTranslate-master.zip) as I see .. I'll have to know the command you're running to generate [ms5.txt_translate](https://github.com/KabueMurage/CL-QuickTranslate/files/9087407/ms5.txt) so I can identify the issue ..  I also recommend to [check with your current code page](https://github.com/KabueMurage/CL-QuickTranslate#known-issues).

**Reply by karubn** [reply]
Thank you for the answer, first of all, Mr. Kabue Murage, there is no problem with the bat file in that zip file anyway, but when I drag the ms4.txt file in the zip to cli.bat, when I open the ms4_translate.txt file and look at the ms4_translate.txt file, the translation is done, but when I drag the ms5.txt file to the cli.bat file It creates the ms5_translate.txt file, but when you open the ms5_tranlate.txt file, there is no translation, a blank page appears.

**Reply by karubn** [reply]
The code inside the bat file I run for ms5.txt (CL-QuickTranslate.exe -f "%1" -l tr -o "%1_translate.txt") I use like this but (CL-QuickTranslate.exe -f "ms5.txt" -l tr -o "ms5_translate.txt") if I do it like this, the ms5_tranlate.txt file comes out as untranslated and empty

**Reply by MurageKabui** [reply]
@karubn , Same behavior here.. I checked with the API and turns out there is now a 5,000 _character_ limit ... 
I trimmed the contents of ms5.txt and it worked .. So you'll have to split the contents of ms5.txt to multiple parts and have it process each part. Even better, make a batch file to loop through ms5-A.txt ms5-B.txt etc. while redirecting the translated text 
to one common file (Using operators >>) . Lemme know if I can help you with the batch script.
